components/AriaTable/AriaTable.stories.tsx

```diff
@@ -155,145 +163,207 @@ Alignment.args = {
+const Customize: ComponentStory<any> = (args) => (
+  <TableForStory
+    css={{ c: '$hiContrast' }}
+    aria-label="People"
+    aria-describedby="basic-table-caption"
+    {...args}
+  >
+    <Caption css={{ c: '$hiContrast' }} id="basic-table-caption">
+      People with some information
+    </Caption>
+    <Thead css={{ c: '$hiContrast' }}>
+      <Tr css={{ c: '$hiContrast' }}>
+        <Th css={{ c: '$hiContrast' }}>first name</Th>
+        <Th>last name</Th>
+        <Th>Status</Th>
+        <Th>Role</Th>
+      </Tr>
+    </Thead>
+    <Tbody css={{ c: '$hiContrast' }}>
+      <Tr>
+        <Td>John</Td>
+        <Td>Doe</Td>
+        <Td>
+          <Badge variant="green">Connected</Badge>
+        </Td>
+        <Td>Developer</Td>
+      </Tr>
+      <Tr>
+        <Td>Johny</Td>
+        <Td>Depp</Td>
+        <Td>
+          <Badge variant="orange">AFK</Badge>
+        </Td>
+        <Td>Actor</Td>
+      </Tr>
+      <Tr>
+        <Td>Natalie</Td>
+        <Td>Portman</Td>
+        <Td>
+          <Badge variant="green">Connected</Badge>
+        </Td>
+        <Td>Actor</Td>
+      </Tr>
+      <Tr>
+        <Td>Luke</Td>
+        <Td>Skywalker</Td>
+        <Td>
+          <Badge variant="red">Disconnected</Badge>
+        </Td>
+        <Td>Star wars</Td>
+      </Tr>
+    </Tbody>
+    <Tfoot css={{ c: '$hiContrast' }}>
+      <Tr>
+        <Td css={{ textAlign: 'center' }}>
+          <Button
+            ghost
+            variant="secondary"
+            css={{ fontSize: '$1', height: '$5', boxShadow: 'none' }}
+          >
+            Load more...
+          </Button>
+        </Td>
+      </Tr>
+    </Tfoot>
+  </TableForStory>
+);
+
+export const Columns: ComponentStory<any> = ({ transform, ...args }) => (
+  <TableForStory aria-label="People" aria-describedby="basic-table-caption" {...args}>
+    <Caption id="basic-table-caption">People with some information</Caption>
+    <Thead>
+      <Tr>
+        <Th transform={transform}>first name</Th>
+        <Th transform={transform}>last name</Th>
+        <Th transform={transform}>Status</Th>
+        <Th transform={transform}>Role</Th>
+      </Tr>
+    </Thead>
+    <Tbody>
+      <Tr>
+        <Td>John</Td>
+        <Td>Doe</Td>
+        <Td>
+          <Badge variant="green">Connected</Badge>
+        </Td>
+        <Td>Developer</Td>
+      </Tr>
+      <Tr>
+        <Td>Johny</Td>
+        <Td>Depp</Td>
+        <Td>
+          <Badge variant="orange">AFK</Badge>
+        </Td>
+        <Td>Actor</Td>
+      </Tr>
+      <Tr>
+        <Td>Natalie</Td>
+        <Td>Portman</Td>
+        <Td>
+          <Badge variant="green">Connected</Badge>
+        </Td>
+        <Td>Actor</Td>
+      </Tr>
+      <Tr>
+        <Td>Luke</Td>
+        <Td>Skywalker</Td>
+        <Td>
+          <Badge variant="red">Disconnected</Badge>
+        </Td>
+        <Td>Star wars</Td>
+      </Tr>
+    </Tbody>
+    <Tfoot>
+      <Tr>
+        <Td colSpan={4} css={{ textAlign: 'center' }}>
```

Comment:
@seedy Please add a story with multiple elements in footer, just to be sure the tweak is not too bad

Reply:
Seems not to work as expected... On it